src/pybind/mgr/telegraf/module.py

```diff
@@ -129,12 +129,7 @@ def get_pg_stats(self):
                     'num_pgs', 'num_objects', 'num_pools']:
             stats[key] = pg_status[key]
 
-        pg_states = ['active', 'peering', 'clean', 'scrubbing', 'undersized',
-                     'backfilling', 'recovering', 'degraded', 'inconsistent',
-                     'remapped', 'backfill_toofull', 'wait_backfill',
-                     'recovery_wait']
-
-        for state in pg_states:
+        for state in PG_STATES:
```

Comment:
The list of states in PG_STATES is much longer than the one used in the Telegraf module before. Not sure if that has any side effects/consequences?